utils.js

```diff
@@ -141,6 +141,14 @@ function createRacks(nrOfTeams, nrOfTokens) {
   });
 }
 
+// Create tokens.....for now it dosnt take any arguments
+function createTokens() {
+  // Our tokes (for now) is just a simple array of { name: letter }
+  'ABCDEFGH'.split('').map((letter) => {
```

Comment:
This needs a `return`. Tip: don't forget you can always copy/paste your functions inside the browser console to you can actually test that they work 👍🏻